package.json

```diff
@@ -22,7 +22,7 @@
     "url": "https://github.com/mapbox/react-native-mapbox-gl"
   },
   "scripts": {
-    "preinstall": "node ./scripts/download-mapbox-gl-native-ios-if-on-mac.js 3.3.1",
+    "preversion": "./scripts/download-mapbox-gl-native-ios.sh 3.3.1",
```

Comment:
Also if the `download-mapbox-gl-native-ios.sh` fails it won't publish new version?